--- Term-Gnuplot-0.90380905/gnuterm/term/linux.trm.orig	2002-07-26 19:42:28.000000000 +0200
+++ Term-Gnuplot-0.90380905/gnuterm/term/linux.trm	2006-10-23 20:54:27.498821500 +0200
@@ -101,8 +101,6 @@
 static int linux_startx, linux_starty, linux_lasty;
 static int linux_angle;
 TERM_PUBLIC int LINUX_graphics_allowed = FALSE; /* also used in vgagl.trm */
-extern void drop_privilege __PROTO((void));
-extern void take_privilege __PROTO((void));
 
 typedef int (*linux_line_func_ptr) __PROTO((int x1, int y1, int x2, int y2));
 
@@ -122,9 +120,6 @@
 
     LINUX_graphics_allowed = FALSE;
 
-    if (geteuid() != 0)
-	return;			/* if we aren't root, we cannot init graphics */
-
     if ((pipe = popen("/usr/bin/tty", "r")) != NULL) {
 	line[0] = 0;
 	fgets(line, 256, pipe);
@@ -148,9 +143,7 @@
 	}
     }
     if (LINUX_graphics_allowed) {
-	take_privilege();
 	vga_init();
-	drop_privilege();
     } else {
 	/* err - shouldn't we give up root uid whatever happens ?
 	 * or perhaps vga_init() does it ?
